README.md

```diff
@@ -1,4 +1,8 @@
 # lotus_salon
 Lotus Salon Project
 
+test!
```

Comment:
looks okay, but seems like you didn't do much

Reply:
yes that was all that was required of me to test it out.